systemtests/src/main/java/io/enmasse/systemtest/operator/OperatorManager.java

```diff
@@ -250,9 +250,8 @@ public void deleteEnmasseAnsible() {
 
     public boolean clean() throws Exception {
         // Delete addressspace CRD first as deleting via label will not wait for finalizers
-        ExecutionResultData result = KubeCMDClient.runOnCluster("delete", "-v", "10", "crd", "addressspaces.enmasse.io");
-        result = KubeCMDClient.runOnCluster("delete", "-v", "10", "crd", "-l", "app=enmasse");
-        if (!result.getRetCode()) {
+        KubeCMDClient.runOnCluster("delete", "-v", "10", "crd", "addressspaces.enmasse.io");
```

Comment:
maybe we could remove the -v 10 , which gives too much vervosity, I think @k-wall added this and I'm not sure if it's not needed anymore

Reply:
Ulf added this line :)